core/src/main/java/org/nervos/ckb/methods/type/cell/Cell.java

```diff
@@ -2,6 +2,16 @@
 
 /** Copyright © 2018 Nervos Foundation. All rights reserved. */
 public class Cell {
-  public CellOutput cell;
+  public CellWithData cell;
   public String status;
+
+  public static class CellWithData {
+    public Data data;
```

Comment:
in ckb called CellData

Reply:
Updated. https://github.com/nervosnetwork/ckb-sdk-java/pull/164/commits/d88ded8fb1193a6335c54750600a68523a69b5c4